config/application.yml.default

```diff
@@ -336,6 +336,7 @@ use_vot_in_sp_requests: true
 usps_auth_token_refresh_job_enabled: false
 usps_confirmation_max_days: 30
 usps_ipp_password: ''
+usps_eipp_client_id: ''
 usps_ipp_client_id: ''
```

Comment:
Should we add some fake default values in the `test:` stanza so we can verify that they're different? So we can make sure that we are passing the correct values (with the same value we have no way to determine which one was passed)

Reply:
Is that a pattern across the app?  I made `usps_eipp_client_id` an empty string because I wanted to keep it consistent with what I saw for `usps_ipp_client_id`.  Curious to hear more, or let me know if there's an example I should follow.

Reply:
now that we have two values, it matters that we want to be able to differentiate them right? like we have multiple placeholder values for `hmac_fingerprinter_key_queue`

Reply:
I'm going to follow up in slack.